packages/@aws-cdk/aws-ecs-patterns/lib/ecs/application-load-balanced-ecs-service.ts

```diff
@@ -131,6 +141,7 @@ export class ApplicationLoadBalancedEc2Service extends ApplicationLoadBalancedSe
       propagateTags: props.propagateTags,
       enableECSManagedTags: props.enableECSManagedTags,
       cloudMapOptions: props.cloudMapOptions,
+      vpcSubnets: props.serviceVpcSubnets,
```

Comment:
we might need to be careful about this since `vpcSubnets` is only allowed when in `AwsVpc` networking mode.